core/src/main/scala/chisel3/Intrinsic.scala

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3._
+import chisel3.experimental.{requireIsChiselType, Param, SourceInfo}
+import chisel3.internal.firrtl.ir._
+import chisel3.internal.{Builder, OpBinding}
+import chisel3.internal.Builder.pushCommand
+
+object Intrinsic {
+
+  /**
+    * Create an intrinsic statement.
```

Comment:
```suggestion
  /** Create an intrinsic statement.
```
ScalaDoc style has the first comment on the first line of the comment (on 2nd line is JavaDoc style, probably will do the right thing but generally we should follow ScalaDoc style).

Same comment on ScalaDoc below

Reply:
Gosh darn it, fired up VSCode specifically to have it generate this for me so I wouldn't accidentally get a detail wrong haha.  Welp.

Sure thing, will fix thanks!